oauthproxy_test.go

```diff
@@ -291,8 +291,7 @@ func TestBasicAuthPassword(t *testing.T) {
 	opts.Validate()
 
 	providerURL, _ := url.Parse(providerServer.URL)
-	const emailAddress = "michael.bland@gsa.gov"
-	const username = "michael.bland"
+	const emailAddress = "john.doe@example.com"
```

Comment:
Should we add a comment somewhere around here that the username is used but the username is equal to the email when the username isn't provided by the provider? Otherwise I think there will be some discrepancy between the tests suggesting the email is used and the code using the username, which happens to equal the email

Reply:
Added as comment where the `expectedHeader` is constructed.